app/assets/javascripts/discourse-common/addon/lib/raw-templates.js

```diff
@@ -0,0 +1,38 @@
+import { getResolverOption } from "discourse-common/resolver";
+
+export const __DISCOURSE_RAW_TEMPLATES = {};
```

Comment:
should we reset this const between tests ?

Reply:
If we reset it all tests will fail! Those templates are not reset.